src/python/ddapp/tabledemo.py

```diff
@@ -248,12 +250,8 @@ def segmentTableObjects(self):
     def graspTableObject(self, side='left'):
 
         obj, objFrame = self.getNextTableObject(side)
-        if self.useCollisionEnvironment:
-            objAffordance = om.findObjectByName(obj.getProperty('Name') + ' affordance')
-
+            
```

Comment:
@iamwolf : i am aware this probably introduces conflicts in tabledemo.py, I'd like to fix them after the merge as this PR is complicated.